terraform/README.md

```diff
@@ -0,0 +1,67 @@
+# Terraform for wire-server
+
+This directory contains (aspires to contain) all the terraform required to setup
+wire-server. The `environment` directory is to be considered the "root"
+directory of terraform.
+
+## How to create a new environment
+
+Recommended: Use nix-shell from the root of this repository to ensure that you
+have the right version of terraform.
+
+Run all commands from `terraform/environment` directory.
+
+1. Export "CAILLEACH_DIR" environment variable to a repository where you want to
+   store environment specific data.
+1. Export "ENV" as the name of the environment
+1. Create environment directory.
+   ```bash
+   export ENV_DIR="$CAILLEACH_DIR/environments/$ENV"
+   mkdir -p "$ENV_DIR"
+   ```
+1. Create backend-config in `"$ENV_DIR/backend.tfvars` which looks like this:
+   ```tf
+   region  = "<aws-region>"
+   bucket  = "<aws-bucket>"
+   key = "<s3-backend-key>"
+   dynamodb_table = "<dynamodb-lock-table>"
+   ```
+
+   Please refer to [s3 backend
+   docs](https://www.terraform.io/docs/backends/types/s3.html) for details.
+1. Create token from hetzner cloud and put the following contents (including the export) 
+    in a file called `$ENV_DIR/hcloud-token.dec`<sup>[1]</sup>:
+```suggestion
```

Comment:
There is an accidental commit of a suggestion pragma here.

Reply:
Good catch, thanks!